Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.10-slim
 
 # Optional build argument for different environments
 ARG ENV
 
 RUN apt-get update -y \
-    && apt-get install --no-install-recommends -y gcc libc-dev musl-dev postgresql-client libpq-dev
+    && apt-get install --no-install-recommends -y gcc libc-dev musl-dev postgresql-client libpq-dev make
```

Comment:
Why did we have to add `make`?

Reply:
Building the image failed due to one of the Python packages failing to compile, complaining that make couldn't be found. Adding this fixed it. It seems that the newer Docker image has different packages included by default (I think it uses a newer version of Debian), which is no big deal.

Reply:
Gotcha. Yeah, I thought we talked about this before with another image. I had deja vu when posting the question. 😆 
Thanks!